index.html

```diff
@@ -18,36 +19,29 @@
     <link href="https://fonts.googleapis.com/css?family=Merriweather:400,700" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css?family=Lato:400,700,400i" rel="stylesheet" type="text/css">
     <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
-    <link href="{{ site.github.url }}/css/transform.css" rel="stylesheet">
-    <link rel="shortcut icon" type="image/x-icon" href="{{ site.github.url }}/favicon.ico">
+    <link href="{{ site.github.url }}/css/index.css" rel="stylesheet">
+    <link rel="shortcut icon" type="image/x-icon" href="{{ site.github.url }}/favicon.svg">
 </head>
 
 <body>
-    <h1>View IRS E-File Forms</h1>
-    <form id="url-form" class="url-form">
-        <h2>Select an XML form to visualize. Supported tax years: 2008 - 2021</h2>
-        <label for="url-input" id="url-label" class="screen-reader">Search</label>
-        <input type="text" id="url-input" placeholder="http://example.com/IRS990.xml">
-        <button type="submit" aria-label="Submit URL to process">
-            <!-- Font Awesome arrow-circle-right -->
-            <svg width="3em" height="3em" viewBox="0 0 1792 1792" xmlns="http://www.w3.org/2000/svg">
-                <title>Submit</title>
-                <path
-                    d="M1413 896q0-27-18-45l-91-91-362-362q-18-18-45-18t-45 18l-91 91q-18 18-18 45t18 45l189 189h-502q-26 0-45 19t-19 45v128q0 26 19 45t45 19h502l-189 189q-19 19-19 45t19 45l91 91q18 18 45 18t45-18l362-362 91-91q18-18 18-45zm251 0q0 209-103 385.5t-279.5 279.5-385.5 103-385.5-103-279.5-279.5-103-385.5 103-385.5 279.5-279.5 385.5-103 385.5 103 279.5 279.5 103 385.5z" />
-            </svg>
-        </button>
-        <p id="url-error"></p>
-        <p>Demo URL: <a href="{{ site.demo_url }}">{{ site.demo_url }}</a></p>
-    </form>
+    <div class="header">
+        <a href="https://nj.gov/"><img class="nj-banner-header-seal" src="img/nj_state_seal.png" alt="NJ flag"/>Official Site of the State of New Jersey</a>
+        <p>Governor Phil Murphy • Lt. Governor Sheila Oliver</p>
+    </div>
```

Comment:
[pebble] Did you consider using the [NJWDS banner](https://newjersey.github.io/njwds/components/detail/banner.html) directly? It seems like you're avoiding importing all the NJWDS css and whatnot here, which I assume is an intentional choice. Even so, I think we want to ensure the banner matches the state banners on other sites. I notice a couple differences, like the lack of the `aria-label`, the usage of the `<section>` semantic html, the "Get Updates" button, etc.

Reply:
I did consider it, but originally decided against it since the repo is using Jekyll and isn't set up with`npm`.

I see the value in matching banners between state sites from a user trust perspective. Have managed to copy over all the CSS/HTML elements necessary to match it in a new commit.

Reply:
Fair, although FYI you can just include NJWDS directly as a built set of CSS and JS, no need for npm.

Reply:
RIP good to know - will do that next time.